weather-forecast/package.json

```diff
@@ -11,6 +12,7 @@
     "@types/react": "^17.0.37",
     "@types/react-dom": "^17.0.11",
     "axios": "^0.24.0",
+    "classnames": "^2.3.1",
```

Comment:
It would be better to use `clsx` package insted.  It has the same functionality by it is more performant.